srcpkgs/vmware-horizon-client/files/vmware-USBArbitrator/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+exec 2>&1
+
+exec chpst -P /usr/bin/vmware-usbarbitrator
```

Comment:
You want to add the `-f` flag at the end to run it in the foreground.
After that the service runs fine, but the usb support in vmware horizon still seems to be broken.

Reply:
Fixed, and USB seems to be working for me!